dxtbx/model/__init__.py

```diff
@@ -9,6 +9,7 @@
 from dxtbx.model.scan import *
 from dxtbx.model.crystal import *
 from dxtbx.model.profile import *
+from dxtbx.model.scaling_model import *
```

Comment:
...could we please not _import *_?

This makes it basically impossible to find out where a name came from and thus incurs a huge technical debt for the unfortunate soul who one day wants to figure out how the stuff in this file works. And that is without any name conflicts.

Reply:
Okay, I think I can remove this and just use an import in the experiment_list.py instead when I import ScalingModelFactory - and then even this can be removed if we use the setuptools?

Reply:
comment was along the lines of 

```python
import dxtbx.model.scaling_model
import otherthing
dxtbx.model.scaling_model.thing() # so that's a scaling_model thing
```
is better than

```python
from dxtbx.model.scaling_model import *
from otherthing import *
thing() # so where does this come from? Even if you know that scaling_model has a thing, does otherthing have a thing that overwrote the thing from scaling_model? Now you have to look into 2 more files to find the answer. Good luck if any of them use import *.
```

If the ```import``` itself is not required then that is obviously even better.